pom.xml

```diff
@@ -62,7 +62,7 @@ under the License.
     <wagonVersion>3.3.2</wagonVersion>
     <securityDispatcherVersion>1.4</securityDispatcherVersion>
     <cipherVersion>1.7</cipherVersion>
-    <modelloVersion>1.10.0</modelloVersion>
```

Comment:
We will have to wait until 1.10.1 is released, we don't use snapshots permanently.

Reply:
as said it's only for CI testing. It's green so I can release modello